stdlib/test/collections/test_inline_list.mojo

```diff
@@ -0,0 +1,94 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+# RUN: %mojo %s
+
+from collections import InlineList, Set
+from testing import assert_equal, assert_false, assert_true, assert_raises
+
+
+def test_mojo_issue_698():
```

Comment:
**Question** Is this GH issue relevant for your new type?  It's not obviously a "yes" for me, since we append in the following test, so I'd just omit this if we can't justify what value it's adding.

Reply:
Indeed, not relevant, I was a bit too quick on the copy-pasting. I removed it.